quinn-proto/src/packet.rs

```diff
@@ -539,25 +541,38 @@ impl PlainHeader {
         }
     }
 
-    fn decode(
+    /// Decode a plain header from given buffer, assuming a fixed-length CID.
+    pub fn decode(
```

Comment:
It looks like this only has a single caller. I don't see a good reason to maintain two variants?